.kokoro/deploy-build-test.sh

```diff
@@ -67,15 +67,15 @@ cd ./github/nodejs-docs-samples/functions/speech-to-speech/functions
 env
 gcloud components update
 gcloud --version
-gcloud beta functions deploy speechTranslate --runtime nodejs8 --trigger-http \
+gcloud functions deploy speechTranslate --runtime nodejs10 --trigger-http \
     --update-env-vars ^:^OUTPUT_BUCKET=playchat-c5cc70f6-61ed-4640-91be-996721838560:SUPPORTED_LANGUAGE_CODES=en,es,fr
 
 echo "Move to the root directory of the repo…"
 cd ${cwd}/github/firebase-android-client
 
 # Copy the Google services configuration file and test values
 cp ${KOKORO_GFILE_DIR}/google-services.json app/google-services.json
-cp ${KOKORO_GFILE_DIR}/speech_translation_test.xml app/src/main/res/values/speech_translation.xml
+# cp ${KOKORO_GFILE_DIR}/speech_translation_test.xml app/src/main/res/values/speech_translation.xml
```

Comment:
Is there a problem with this file? If so, we should fix it instead of commenting it out.

Reply:
Updated per our conversation.

Reply:
Updated the file in our testing environment.